Homework_9_Incapsulation_Inheritance_Polymorphism/My_Company/src/com/example/employees/Employee.java

```diff
@@ -0,0 +1,78 @@
+package com.example.employees;
+
+public class Employee {
+
+    private String name;
+    private int age;
+    private int salary;
+    private static int minSalary = 2000;
+
+    public Employee(String name, int age, int salary) {
+        this.name = name;
+        setAge(age);
+        setSalary(salary);
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        if (name != null) {
+            this.name = name;
+        }
+    }
+
+    public int getAge() {
+        return age;
+    }
+
+    public void setAge(int age) {
+        if (age > 18) {
+            this.age = age;
+        } else {
+            System.out.println("Please enter valid age");
+        }
+    }
+
+    public int getSalary() {
+        return salary;
+    }
+
+    public void setSalary(int salary) {
+        if (salary > minSalary) {
+            this.salary = salary;
+        } else {
+            System.out.println("The salary isn't correct");
```

Comment:
Better UX -> if the user makes mistake -> we tell them explicitly what is their mistake and how they can fix it

`"The salary should be at least " + MIN_SALARY + ". Please add a valid salary value"`